src/index.html

```diff
@@ -1,13 +1,339 @@
+            <div class="features__description-title">
+              Range & Integrated lights
+            </div>
+            <div class="features__description-text">
+              The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+            </div>
+          </div>
+        </div>
+        <div class="features">
+          <div class="features__photos">
+            <div class="grid">
+              <div class="grid__item grid__item--desktop-1-2">
+                <div class="features__photo-container">
+                  <img
+                    src="./images/photos/details-5.png"
+                    alt="details-1"
+                    class="features__photo"
+                  >
+                </div>
+              </div>
+              <div class="grid__item grid__item--desktop-3-6">
+                <div class="features__photo-container">
+                  <img
+                    src="./images/photos/details-6.png"
+                    alt="details-2"
+                    class="features__photo"
+                  >
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="features__description">
+            <div class="features__description-title">
+              Hydraulic disc brakes & Lightweight
+            </div>
+            <div class="features__description-text">
+              Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contact-us" id="contact-us">
+        <a href="#" class="button__explore button">
+          Explore
+        </a>
+        <h2 class="contact-us-title">
+          Contact us
+        </h2>
+
+        <div class="contact-us__grid grid grid--desktop">
+          <div class="grid__item grid__item--1-5 contact-us__grid--form">
+            <div class="contact-us__form">
+              <form action="/" method="get">
+                <input
+                  name="name"
+                  type="text"
+                  class="contact-us__field form-field"
+                  placeholder="Name"
+                  required
+                  >
+                <input
+                  name="email"
+                  type="email"
+                  class="contact-us__field form-field"
+                  placeholder="E-mail"
+                  required
+                >
+                <textarea
+                  name="message"
+                  class="contact-us__field form-field form-field--textarea"
+                  placeholder="Message"
+                  required
+                ></textarea>
+                <button type="reset" class="button">
```

Comment:
use js for resetting (go to qna if don’t understand how to implement it)

```suggestion
                <button type="submit" class="button">
```